src/mongocxx/options/find.cpp

```diff
@@ -211,6 +214,179 @@ const stdx::optional<class read_preference>& find::read_preference() const {
     return _read_preference;
 }
 
+find& find::convert_all_modifiers() {
+    if (!_modifiers) {
+        return *this;
+    }
+
+    for (auto&& ele : _modifiers->view()) {
+        if (ele.key() == stdx::string_view("$comment")) {
```

Comment:
Would `ele.key() == "$comment"` work, since string_view is supposed to be implicitly constructible from a `const char *`?